### PR TITLE
[hotfix] [docs] Fix typos in State Backends doc

### PR DESCRIPTION
**Pull request by casidiablo**

### CODE REVIEW

File: docs/ops/state/state_backends.md [hunk 1]
```diff
@@ -110,10 +110,10 @@ Limitations of the RocksDBStateBackend:
 
 The RocksDBStateBackend is encouraged for:
 
-  - Jobs with very large state, long windows, large key/value states.
+  - Jobs with a very large state, long windows, large key/value states.
```
Comment (tzulitai) [inline docs/ops/state/state_backends.md]: I'm not entirely sure that this change is necessary, but may be wrong as I'm not a native English speaker :)

**Comment by tzulitai** [comment]
Changes LGTM, minus one comment.

File: docs/ops/state/state_backends.md (hunk 1 above)
Comment (alpinegizmo) [reply docs/ops/state/state_backends.md]: This change is ungrammatical, but the rest are good.

**Comment by tzulitai** [comment]
Thanks! Merging this ..

**Pull request closed without merging**